src/gapfill.c

```diff
@@ -10,6 +10,20 @@
 #include "cross_module_fn.h"
 #include "compat/compat.h"
 #include "export.h"
+#include "gapfill.h"
+
+#define GAPFILL_PATH_NAME "GapFill"
```

Comment:
Nit: This is probably better to have in the header file.

Reply:
Thanks for the review. The define is moved.